arch/arm/boot/dts/overlays/fancom-whc-overlay.dts

```diff
@@ -0,0 +1,191 @@
+/*
+ * vc4-kms-dsi-sn65dsi8x-overlay.dts
+ */
+
+/dts-v1/;
+/plugin/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/bcm2835.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+#include <dt-bindings/clock/bcm2835.h>
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+
+#include "cma-overlay.dts"
+
+/ {
+	compatible = "brcm,bcm2835";
+
+	fragment@0 {
+		target = <&gpio>;
+		__overlay__ {
+			i2c0_pins: i2c0 {
+				brcm,pins = <44 45>;
+				brcm,function = <BCM2835_FSEL_ALT2>;
+				brcm,pull = <BCM2835_PUD_UP>;
+			};
+
+			gpio_restore: gpio_restore {
```

Comment:
I'm not sure that we need to specify all these pins here since it is the default function of these pins. We also didn't specify the GPIOs that are attached to the r/g/b leds here, if we want to be complete, then these should also be mentioned here.